ui/docs/CodeStyle.md

```diff
@@ -0,0 +1,23 @@
+# Code style
+
+<To be added when liniting in pace>
```

Comment:
Is this supposed to say <To be added when **linting** in **place**>. Also what is being added?

Reply:
Linting is in place! Will sort this out